src/main/kotlin/uk/gov/dluhc/printapi/exception/ExplainerDocumentNotFoundException.kt

```diff
@@ -0,0 +1,3 @@
+package uk.gov.dluhc.printapi.exception
+
+class ExplainerDocumentNotFoundException(message: String) : RuntimeException(message)
```

Comment:
This exception class is a rename of `TemporaryCertificateExplainerDocumentNotFoundException` (shown below as a delete); and I've also change the constructor args to simply take a message, rather than an eroId and gssCode as before (the high-order lambda function mentioned above generates the exception message)